lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -62,6 +62,7 @@ $-separator-color: var(--nx-swatch-indigo-40);
   margin: 0;
 
   &:focus {
+    border-width: 2px;
```

Comment:
This results in the height of the button increasing.  You'll need to adjust the padding (subtracting an additional pixel beyond what's already being subtracted) in order to avoid that.

Reply:
It is fixed here 780f81ed834dcaf8c3b44c9e0b360aa37e0c8e79